ITW Sabrina ✨ (#409)
Elao/elao_: content/blog/elao/sabrina-alternante-en-integration.md

Après avoir explorer en autodidacte pendant plusieurs mois, j'ai intégrer une école et compléter une formation socle de trois mois centrée sur l'apprentissage de l'environnement JavaScript et la réalisation de projets.

## Qu'est ce qui t'a donné envie de faire du web ?

J’ai d’abord commencé par compléter le maximum de cours sur Khan Academy avant de décider de me former et d’en faire mon métier. En prenant le temps d’explorer la programmation informatique à mon rythme, j’ai compris que c’était un domaine tout aussi créatif que le cinéma ou le théâtre. De plus, les technologies et les outils que nous utilisons changent constamment, ce qui me permet d'être continuellement en apprentissage.

Ayant vécu une grande partie de ma vie dans divers pays africains, j'ai été sensible au fait que les femmes doivent surmonter de nombreux obstacles rien que pour avoir l'opportunité d'étudier. Plus important encore, le domaine de la technologie et l'informatique sont des domaines qui sont rarement associés aux femmes. C'est aussi avec un devoir d'émancipation et de progrès social que je me lance dans le web. J'espère inspirer, casser les codes et motiver d'autres à se lancer.

> Plus sérieusement, qui ne hurle pas de joie lorsque son code s’exécute à la perfection?🤓

Reviewer: ```suggestion
Plus sérieusement, qui ne hurle pas de joie lorsque son code s’exécute à la perfection? 🤓
```

<video src="https://media.giphy.com/media/55zYS1qxfTXFe/giphy.mp4?cid=ecf05e47lmw7xgbjs2pislcy4b0k5rh5mf4gx94lgzvrxc0p&rid=giphy.mp4&ct=g" autoplay loop></video>

> 

Reviewer: ```suggestion
```

## Qu'est-ce que tu fais au quotidien chez Elao ?

Généralement, je reçois une maquette de la part de Amélie ou de Christelle, que je vais ensuite intégrer avec du HTML et du CSS.
Je dois faire attention à respecter la charte graphique fournie lors de cette étape.

Ensuite, je m’assure de la validité du code HTML et CSS ainsi que de sa compatibilité avec tous le navigateurs et formats d’écran ( les ordinateurs de bureau, les smartphones, les tablettes, etc…).

> On aime travailler en équipe chez Elao, alors je sollicite l'aide de Amélire pour peaufiner l’intégration et vérifier l’accessibilité avant de passer la main aux développeurs qui s’occuperont de câbler tout ça avec des données.

Author: ```suggestion
On aime travailler en équipe chez Elao, alors je sollicite l'aide de Amélie pour peaufiner l’intégration et vérifier l’accessibilité avant de passer la main aux développeurs qui s’occuperont de câbler tout ça avec des données.
```

Reviewer: oui ? 🤓

### Et quand tu ne travailles pas ?

Lorsque je ne travaille pas, j’aime faire des activités qui s’éloignent le plus possible de l’utilisation d’un écran.

> Je fais du yoga régulièrement, et mes collègues peuvent parfois m’apercevoir méditer dans un coin du bureau. Je chante souvent, mais surtout lorsque personne ne m’observe ( je suis un peu timide 🙈).

Reviewer: ```suggestion
Je fais du yoga régulièrement, et mes collègues peuvent parfois m’apercevoir méditer dans un coin du bureau. Je chante souvent, mais surtout lorsque personne ne m’observe (je suis un peu timide 🙈).
```

Je me suis récemment remise à dessiner car j’aime utiliser mes mains et divers outils (crayon, stylo, aquarelle, etc...) pour créer.

![](images/posts/2021/interview/sab-ludo-dessin.jpg)

## L'interview :fast-parrot:

### Tes 3 emojis préf du Slack

Mon émoji préféré est "amaze face", il représente l'étonnement et je l'utilise souvent parce que j'apprends plein de choses avec l'équipe d'Elao.

> Mes deux autres émojis préferés font référence à des blagues internes que je ne peux divulguer. It's top secret.

Reviewer: 🌝

<video src="https://media.giphy.com/media/NdKVEei95yvIY/giphy.mp4?cid=ecf05e47p9x7qmntm1wr78h6jl7rea8gbss2tkt2h0jkqvvj&rid=giphy.mp4&ct=g" autoplay loop></video>

### Les 3 trucs qui traînent toujours autour de ton bureau

> On peut toujours trouver une bouteille d'eau, un éléphant en perles, une petite plante, des posts-its et des stylos.

Reviewer: ```suggestion
On peut toujours trouver une bouteille d'eau, un éléphant en perles, une petite plante, des post-it et des stylos.
```
